planner/core/logical_plan_builder.go

```diff
@@ -2698,19 +2702,123 @@ func (b *PlanBuilder) buildProjectionForWindow(p LogicalPlan, expr *ast.WindowFu
 	return proj, propertyItems[:lenPartition], propertyItems[lenPartition:], newArgList, nil
 }
 
+func (b *PlanBuilder) buildFrameBound(spec *ast.WindowSpec, orderByItems []property.Item, boundClause *ast.FrameBound) (*FrameBound, error) {
```

Comment:
What about keeping its prefix the same with `buildWindowFunctionFrame`?
Change this one two `buildWindowFunctionFrameBound` or change this two to `buildWindowFram` and `buildWindowFrameBound`.